sdk/include/cheri.hh

```diff
@@ -546,54 +546,67 @@ namespace CHERI
 				return *this;
 			}
 
+			private:
+			BoundsProxy &set_inexact_at_most_slow(size_t bounds)
+			{
+				ptraddr_t newBaseAddress =
+				  static_cast<ptraddr_t>(reinterpret_cast<uintptr_t>(ptr()));
```

Comment:
Can we use the address get intrinsic instead of these casts?

Reply:
Sure.  Out of curiosity, why is `PropteryProxyBase` `private` and not `protected`?  `cap.address()` seems nicer than `__builtin...`?

Reply:
Cap.address is fine too.

It’s private because I didn’t see a use case for subclasses accessing it, it’s an implementation detail.